solana/indexer/solana_indexer.go

```diff
@@ -78,23 +80,42 @@ func New(config config.Config) *SolanaIndexer {
 	})
 
 	s := &SolanaIndexer{
-		rpcClient:  rpcClient,
-		grpcClient: grpcClient,
-		logger:     logger,
-		config:     config,
-		pool:       pool,
+		rpcClient:   rpcClient,
+		grpcClient:  grpcClient,
+		logger:      logger,
+		config:      config,
+		pool:        pool,
+		flushTicker: time.NewTicker(time.Second * 15),
```

Comment:
did you put flushticker here so u can tap into it with other stuff?

Reply:
it's so it can be closed on `Close()`